make/scripts/version-info.py

```diff
@@ -139,7 +148,7 @@ def _get_authors_since_release(self):
 
             self._exec('shortlog -s {}..HEAD'.format(last_rel['ref']))
             if self._rc == 0:
-                self._authors_since_release = sorted([l.strip().split(None, 1)[1] for l in self._out.splitlines()], key=str.lower)
+                self._authors_since_release = sorted([l.strip().split(None, 1)[1] for l in self._out.splitlines()])
```

Comment:
You have explicitly decoded `self._out` to string in `_exec` so the string sort should should be operating on Python `str` type in both 2 & 3? Tested on 2.7.12 and 3.5.2 and both worked fine.